akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -381,8 +430,15 @@ private[remote] class ArteryTransport(_system: ExtendedActorSystem, _provider: R
 
   val (afrFileChannel, afrFlie, flightRecorder) = initializeFlightRecorder()
 
+  def createFlightRecorderEventSink(): EventSink = {
+    // FIXME there is some concurrency issue with the FlightRecorder, when shutting down.
+    //       It crashes the JVM.
+    // flightRecorder.createEventSink()
+    IgnoreEventSink
```

Comment:
Maybe we can wrap access to this sink in a synchronized block, it is used *only* for startup/shutdown events anyway